fds/services/git_service.py

```diff
@@ -11,12 +11,24 @@ class GitService(object):
     Git Service responsible for all the git commands of fds
     """
     def __init__(self):
-        self.repo_path = os.path.curdir
+        self.repo_path = self.get_repo_path()
         self.printer = PrettyPrint()
 
+    @staticmethod
+    def get_repo_path():
+        try:
+            path_cmd = execute_command(["git", "rev-parse", "--show-toplevel"], capture_output=True)
+            repo_path = convert_bytes_to_string(path_cmd.stdout).strip()
+        except Exception:
+            repo_path = os.path.curdir
```

Comment:
Again, shouldn't we check that we don't get an unexpected error?

Reply:
Resolved